segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -114,6 +117,32 @@ public static void recoverAllSegments(Storage storage, Map<Integer, DebugStreamS
         Futures.allOf(futures).get(TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
     }
 
+    public static CompletableFuture<Void> backUpMetadataAndAttributeSegments(Storage storage, DebugStreamSegmentContainer container,
+                                                                              ExecutorService executorService) {
+        Preconditions.checkNotNull(storage);
+        String metadataSegmentName = NameUtils.getMetadataSegmentName(container.getId());
+        String attributeSegmentName = NameUtils.getAttributeSegmentName(metadataSegmentName);
+        String timeStamp = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss").format(new Date());
+        return container.copySegment(storage, metadataSegmentName, "NewMetadataSegment_" + timeStamp + "_" +
+                container.getId(), executorService)
+                .exceptionally(e -> {
+                    if (Exceptions.unwrap(e) instanceof StreamSegmentNotExistsException) {
+                        log.info("Segment '{}' doesn't exist.", metadataSegmentName);
+                        return null;
+                    }
+                    log.error("Error occurred while copying the segment.");
+                    return null;
+                }).thenAcceptAsync(x -> container.copySegment(storage, attributeSegmentName, "NewAttributeSegment" + timeStamp + "_" +
+                        container.getId(), executorService)).exceptionally(e -> {
+                    if (Exceptions.unwrap(e) instanceof StreamSegmentNotExistsException) {
```

Comment:
Please indent this block , so that it is obvious that if statement is inside exceptionally.